boards/nucleo_f429zi/Makefile

```diff
@@ -8,6 +8,10 @@ include ../Makefile.common
 OPENOCD=openocd
 OPENOCD_OPTIONS=-f openocd.cfg
 
+APP=../../../libtock-c/examples/lcd_16x2/build/cortex-m4/cortex-m4.tbf
+KERNEL=$(CWD)/target/$(TARGET)/debug/$(PLATFORM).elf
+KERNEL_WITH_APP=$(CWD)/target/$(TARGET)/debug/$(PLATFORM)-app.elf
+
```

Comment:
I don't think we want to merge any changes to the nucleo Makefile, not everyone will have libtock-c in that relative directory, for example.

Reply:
Yes, sorry. I modified the Makefile.